overlord/snapstate/handlers.go

```diff
@@ -119,6 +119,28 @@ func snapSetupAndState(t *state.Task) (*SnapSetup, *SnapState, error) {
 const defaultCoreSnapName = "core"
 const defaultBaseSnapsChannel = "stable"
 
+func inFlight(st *state.State, name string) (bool, error) {
```

Comment:
The function name and its arguments give no hint about what this is doing. Perhaps changeInFlight, and snapName?